cmd/root.go

```diff
@@ -39,6 +39,9 @@ var logger log.Logger = &log.StdoutLogger{
 	LogLevel: log.Debug,
 }
 
+// name of the executable, this is for the help messages
+var ExecutableName string = os.Args[0]
```

Comment:
so if this arg has paths it will show up...

Reply:
That's true, but it's intentional based on how it's being used. For example:

```
% ./path/to/exec/ff

To get started, run: ./path/to/exec/ff init
Optional: Set the FIREFLY_HOME environment variable for FireFly stack configuration path.

Usage:
  ./path/to/exec/ff [command]

Available Commands:
  accounts    Work with accounts in a FireFly stack
  completion  Generate the autocompletion script for the specified shell
  deploy      Deploy a compiled smart contract
  docs        Generate markdown documentation for all commands
  help        Help about any command
  info        Get info about a stack
  init        Create a new FireFly local dev stack
  list        List stacks
  logs        View log output from a stack
  ps          Returns information on running stacks
  pull        Pull a stack
  remove      Completely remove a stack
  reset       Clear all data in a stack
  start       Start a stack
  stop        Stop a stack
  upgrade     Upgrade a stack to a different version
  version     Prints the version info

Flags:
      --ansi string   Control when to print ANSI control characters ("never"|"always"|"auto") (default "auto")
  -h, --help          Help for ./ff/ff
  -v, --verbose       Verbose log output

Use "./path/to/exec/ff [command] --help" for more information about a command.
```

As you can see, the executable was built locally in a subpath, so it makes sense for the help messages to reflect the subpath. This way, users can simply copy and paste the command from the help message without needing to adjust the path manually.

Reply:
awesome cool